zeppelin-web/karma.conf.js

```diff
@@ -67,6 +67,7 @@ module.exports = function(config) {
       'bower_components/ace-builds/src-noconflict/keybinding-emacs.js',
       'bower_components/ace-builds/src-noconflict/ext-language_tools.js',
       'bower_components/ace-builds/src-noconflict/theme-chrome.js',
+      'bower_components/ace-builds/src-noconflict/mode-javascript.js',
```

Comment:
Is this related to this PR? Otherwise, could you please explain what it is?

Reply:
It is not related with removing pig. It was automatically added when building Zeppelin, so I just included in the pr.

Reply:
It doesn't seem to make any issue but it's a bit weird that it was added while building it. Could you please check it again? If it does, please let me know how to reproduce it.

Reply:
I tested on Windows and Linux, the result was same.
1. Clone apache/zeppelin
2. Build with command 'mvn clean package -DskipTests -Phadoop2 -X'

![capture](https://user-images.githubusercontent.com/38900338/180728906-6b447e81-08e2-455a-96c5-9953a5efa041.png)